### PR TITLE
Tweak nginx.

### PR DESCRIPTION
**Pull request by bratao**
I used some tested optimization that I use in my server.

Is this a valid contribution ??

### CODE REVIEW

File: frameworks/PHP/php-laravel/deploy/nginx.conf [hunk 1]
```diff
@@ -1,7 +1,9 @@
 worker_processes  8;
 
 events {
-    worker_connections  1024;
+    worker_connections 2048;
+	multi_accept on;
```
Comment (methane) [inline frameworks/PHP/php-laravel/deploy/nginx.conf]: Indent

**Comment by hamiltont** [comment]
This seems to be acceptable within our rules. 

Pinging @msmith-techempower / @LadyMozzarella for a second opinion - look good to you?

**Comment by msmith-techempower** [comment]
It's just for `laravel`? That's fine with me. LGTM

**Comment by Dith3r** [comment]
we could also consider adding worker_rlimit_nofile    30000;

**Comment by hamiltont** [comment]
@Dith3r feel free to send in a one-off PR for that!

**Pull request merged**